src/background/onboarding.js

```diff
@@ -0,0 +1,35 @@
+import browser from "webextension-polyfill";
+import SLStorage from "../popup/SLStorage";
+
+let POST_SETUP_URL = null;
+
+function initService() {
+  browser.runtime.onInstalled.addListener(async function () {
+    const hasFirstRun = await SLStorage.get(SLStorage.SETTINGS.HAS_FIRST_RUN);
+    if (hasFirstRun) return;
+
+    browser.tabs.create({
```

Comment:
I think we need to add an `await` in front of this call. If you use Pycharm (or Webstorm), it will flag these warnings automatically :).

Reply:
OK for adding `await`. In fact, I don't use `await` here because these 2 actions can be run in parallel.

Reply:
If 2 functions can be run in parallel, should we use `Promise.all` here?

Reply:
For now we don't need. We should only use `Promise.all` if we need to know the returned value of many promises.